tests/templates/kuttl/smoke/40-install-hdfs.yaml.j2

```diff
@@ -23,20 +23,38 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+      resources:
```

Comment:
2x times below as well:

My recommendation would be to remove this lines, so that we implicitly test the default hdfs resources

Reply:
done